app/web3/transactionConfirmation.ts

```diff
@@ -0,0 +1,73 @@
+/* tslint:disable:no-console */
+
+import config from "../config";
+import web3Provider from "./web3Provider";
+
+export const transactionConfirmation = async (
+  transactionHash: string,
+  transactionMinedCallback: any,
+  newBlockCallback: any
+) => {
+  console.log("waiting for transaction: " + transactionHash);
```

Comment:
i am not sure if leaving console logs is good idea. We would need to strip them in prod build but it's kinda hard to do without babel plugin :/

Reply:
Those were left intentionally as for sure I will make changes in this file during this week so they can be still useful. But I agree at least I could comment them out.

Reply:
okay i think you can leave them as it is now. Just remember to remove them later.